config/ini.go

```diff
@@ -97,9 +96,11 @@ func (ini *IniConfig) parseFile(name string) (*IniConfigContainer, error) {
 		}
 		if bComment != nil {
 			line = bytes.TrimLeft(line, string(bComment))
-			line = bytes.TrimLeftFunc(line, unicode.IsSpace)
+			// Need append to a new line if multi-line comments.
+			if comment.Len() > 0 {
+				comment.WriteByte('\n')
```

Comment:
Don't  add a new line after the  last comment  . e.g:
```ini
#Com1
#Com2
appid = beego
```
the old way result (last empty line)
```
#Com1
#Com2

appid = beego
```
So I change the way of appending new line.  Add new line before the comment for non-first comment.